examples/plugin-compat/rspack.config.js

```diff
@@ -4,6 +4,7 @@ const CopyPlugin = require("copy-webpack-plugin");
 const HtmlPlugin = require("@rspack/plugin-html").default;
 const { StatsWriterPlugin } = require("webpack-stats-plugin");
 const minifyPlugin = require("@rspack/plugin-minify");
+const manifestPlugin = require("rspack-manifest-plugin").WebpackManifestPlugin;
```

Comment:
looks a bit strange

Reply:
I may make webpack-manifest-plugin one day, but in order to make minimal change so I don't change internal export variable

Reply:
get👌🏻